recipes/dydxprotocol-v4-client-py/meta.yaml

```
@@ -0,0 +1,106 @@
+{% set name = "dydxprotocol-v4-client-py" %}
+{% set version = "1.1.31" %}
+
+package:
+  name: dydxprotocol-v4-client-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/dydxprotocol/v4-clients/archive/refs/tags/v4-client-js@{{ version }}.tar.gz
+  sha256: eb6ffc4e53aa0a22f195092f2102fb5233c5c4dda7b11700208a1974d1428297
+  patches:
+    - patches/0001-update-v4-proto-version.patch
+    - patches/0002-use-pyext-over-_ubp.patch
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - cd v4-client-py-v2 && poetry build -f wheel
+
+requirements:
+  host:
+    - dydxprotocol-v4-proto-py ==5.0.6
+    - poetry
+    - python >=3.9
+    - typing-extensions ==4.12
```

What's the reason to have this block?
```suggestion
requirements:
  host:
    - dydxprotocol-v4-proto-py ==5.0.6
    - poetry
    - python >=3.9
    - typing-extensions ==4.12
```